CHANGELOG.md

```diff
@@ -12,11 +54,11 @@ x.y.z Release notes (yyyy-MM-dd)
 * Add support for embedded objects. Embedded objects are objects which are
   owned by a single parent object, and are deleted when that parent object is
   deleted. They are defined by subclassing `EmbeddedObject` /
-  `RLMEmbeddedObject` rather than `Object` / `RLMObject`.
+	`RLMEmbeddedObject` rather than `Object` / `RLMObject`.
+* Add support for next generation sync. Next generation sync includes the
```

Comment:
"Next gen sync" is not too descriptive. Perhaps something like:
"Support for syncing to MongoDB instead of Realm Object Server. Applications must be created at
  realm.mongodb.com"

Reply:
I would put this at the top of the enhancements.